components/tikv_alloc/src/lib.rs

```diff
@@ -112,9 +113,10 @@ pub use self::imp::*;
 // The implementation of this crate when jemalloc is turned on
 #[cfg(all(unix, not(fuzzing), not(feature = "no-jemalloc")))]
 mod imp {
+    use jemalloc_ctl::{stats, Epoch};
```

Comment:
`Epoch` can be confused with raft epoch, how about 
`use jemalloc_ctl::{stats, Epoch as JeEpoch}` ?

Reply:
fixed